engines/tensorrt/build.gradle

```diff
@@ -31,14 +33,14 @@ processResources {
                 new URL("${url}/${entry.key}").withInputStream { i -> file.withOutputStream { it << i } }
             }
         }
-        file("${project.buildDir}/classes/java/main/native/lib").mkdirs()
+        file("${baseResourcePath}/native/lib").mkdirs()
         copy {
             from jnilibDir
-            into "${project.buildDir}/classes/java/main/native/lib"
+            into "${baseResourcePath}/native/lib"
         }
 
         // write properties
-        def propFile = file("${project.buildDir}/classes/java/main/native/lib/tensorrt.properties")
+        def propFile = file("${baseResourcePath}/native/lib/tensorrt.properties")
         propFile.text = "version=${trt_version}-${version}"
```

Comment:
Did you verify whether changes to the versions specified here will trigger task execution?

Reply:
Fixed - added input properties